frontend/src/app/store/app.store.ts

```diff
@@ -3,11 +3,12 @@ import { Task } from "../models/task";
 import { Injectable } from "@angular/core";
 import { BehaviorSubject, Observable } from "rxjs";
 import { TaskFilters } from "../models/task-filters";
+import { TaskStatus } from "../models/task-status";
 
 
 @Injectable({ providedIn: 'root' })
 export class AppStore {
-  private state: AppState = { filters: { status: null }, tasks: [] };
+  private state: AppState = { filters: { status: null , searchString: null }, tasks: [] };
```

Comment:
Now it has one extra space after `status: null` and before comma. Here is how it should be: 
```typescript
private state: AppState = { filters: { status: null, searchString: null }, tasks: [] };
```

Reply:
Done